tests/connector-service-tests/test/testkit/securedconnectorclient.go

```diff
@@ -72,6 +73,17 @@ func (cc securedConnectorClient) RenewCertificate(t *testing.T, url string, csr
 	return &certificateResponse, errorResp
 }
 
+func (cc securedConnectorClient) RevokeCertificate(t *testing.T, url string, csr string) *http.Response {
+	request, err := http.NewRequest(http.MethodPost, url, bytes.NewBuffer([]byte{}))
+	require.NoError(t, err)
+	request.Header.Add("Certificate-Header", csr)
```

Comment:
As discussed, we should not set header here

Reply:
Done!